packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperSynapsePlot.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useMemo } from 'react'
+import { SynapsePlot } from '../Plot'
+import { QueryContextType, useQueryContext } from '../QueryContext'
+import {
+  QueryWrapperPlotNavCustomPlotParams,
+  SynapsePlotWidgetParams,
+} from '../Plot/SynapsePlot'
+import {
+  FacetColumnRequest,
+  QueryFilter,
+  Row,
+} from '@sage-bionetworks/synapse-types'
+import { Typography } from '@mui/material'
+
+export type QueryWrapperSynapsePlotProps = Pick<
+  QueryWrapperPlotNavCustomPlotParams,
+  'onCustomPlotClick'
+> &
+  Omit<SynapsePlotWidgetParams, 'selectedFacets' | 'additionalFilters'>
+
+export type QueryWrapperSynapsePlotRowClickEvent = {
+  row: Row
+  queryContext: QueryContextType
+}
+
+/**
+ * This component wraps a SynapsePlot.  It passes in the current query context, and has a custom
+ * "title" style so that it looks like it belongs in the QueryWrapperPlotNav plots.
+ * @param props
+ * @returns
+ */
+export default function QueryWrapperSynapsePlot(
+  props: QueryWrapperSynapsePlotProps,
+) {
+  const queryContext = useQueryContext()
+  const { currentQueryRequest } = queryContext
+  const { title, onCustomPlotClick } = props
+
+  const widgetParamsMapped: SynapsePlotWidgetParams = useMemo(() => {
+    return {
+      ...props,
+      title: undefined, // we are handling the plot title here in this component
+      showlegend: 'false',
+      displayModebar: 'false',
+    }
+  }, [props])
```

Comment:
as suggested above, if the `widgetParamsMapped` interface was only used in the wiki, we wouldn't need to put booleans in strings.

You could also consider adding color/theming as is done in `FacetNavPanel`, but I'm not sure if there's gonna be a design pass on this after it gets into the staging portal.